client/components/date-range/index.js

```diff
@@ -649,9 +655,17 @@ export class DateRange extends Component {
 			'date-range__picker': true,
 		};
 
+		const calendarInitialDate =
+			this.props.firstSelectableDate ||
+			this.props.firstSelectableDate ||
```

Comment:
I'm repeating `this.props.firstSelectableDate` and this was just a copy-pasta error. Will clean up soon.